boards/esp32-s3-wroom-1-n16r8.json

```diff
@@ -0,0 +1,50 @@
+{
+    "build": {
+        "arduino": {
+            "ldscript": "esp32s3_out.ld",
+            "memory_type": "qio_opi",
+            "partitions": "default_16MB.csv"
+        },
+        "core": "esp32",
+        "extra_flags": [
+            "-DBOARD_HAS_PSRAM",
+            "-DARDUINO_ESP32_S3_N16R8",
+            "-DARDUINO_USB_MODE=1",
+            "-DARDUINO_USB_CDC_ON_BOOT=1",
+            "-DARDUINO_RUNNING_CORE=1",
+            "-DARDUINO_EVENT_RUNNING_CORE=1"
+        ],
+        "f_cpu": "240000000L",
+        "f_flash": "80000000L",
+        "flash_mode": "qio",
+        "hwids": [
+            [
+                "0x303A",
+                "0x1001"
+            ]
+        ],
+        "mcu": "esp32s3",
+        "variant": "esp32_s3_wroom_1_n16r8"
```

Comment:
Using this board file in my own projects let `#include <Arduino.h>` lead to this error:
`framework-arduinoespressif32/cores/esp32/esp32-hal-gpio.h:29:10: fatal error: pins_arduino.h: No such file or directory`.

If I replace the variant name `esp32_s3_wroom_1_n16r8` by `esp32s3`, my projects compile!

Reply:
Same behaviour here, trying to compile for Firebeetle2 N16R8; not sure if by setting it as `esp32s3` we are "missing some cool optimization" or so